### PR TITLE
add myself as reviewer and approver

### PR DESCRIPTION
**Pull request by ipochi**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -4,10 +4,12 @@ reviewers:
   - dberkov
   - jefftree
   - tallclair
+  - ipochi
```
Comment (cheftako) [inline OWNERS:7]: Fully in support of adding you as a reviewer.

File: OWNERS [hunk 2]
```diff
@@ -4,10 +4,12 @@ reviewers:
...
 approvers:
   - anfernee
   - cheftako
   - jkh52
+  - ipochi
```
Comment (cheftako) [inline OWNERS:12]: K/K [policy](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-2) is that approvers have to have first spent 3 months (a release) as a reviewer before becoming an approver.

**Comment by jkh52** [comment]
@ipochi looks like you need to join [Kubernetes membership](https://github.com/kubernetes/community/blob/master/community-membership.md#member) to become a reviewer. One of us can be your sponsor, if you would like.

**Comment by ipochi** [comment]
> @ipochi looks like you need to join [Kubernetes membership](https://github.com/kubernetes/community/blob/master/community-membership.md#member) to become a reviewer. One of us can be your sponsor, if you would like.

Created a PR for the same
https://github.com/kubernetes/org/issues/4368

**Comment by ipochi** [comment]
> > @ipochi looks like you need to join [Kubernetes membership](https://github.com/kubernetes/community/blob/master/community-membership.md#member) to become a reviewer. One of us can be your sponsor, if you would like.
> 
> Created a PR for the same [kubernetes/org#4368](https://github.com/kubernetes/org/issues/4368)

PR merged 🥳

**Pull request merged**